src/promptflow-devkit/tests/sdk_cli_test/unittests/test_flow.py

```diff
@@ -82,3 +82,26 @@ def test_multiple_flex_load(self):
     def test_specify_flow_load(self):
         load_flow(EAGER_FLOWS_DIR / "multiple_flow_yaml" / "flow.dag.yaml")
         load_flow(EAGER_FLOWS_DIR / "multiple_flow_yaml" / "flow.flex.yaml")
+
+    def test_flow_path_not_exist(self):
+        flow_path = EAGER_FLOWS_DIR / "flow_path_not_exist"
+        with pytest.raises(UserErrorException) as e:
+            load_flow(flow_path)
```

Comment:
what's the error message if we passed a folder without yaml in it here?

Reply:
Please see this test: **def test_flow_file_not_exist2(self):**